src/amber/router/router.cr

```diff
@@ -29,9 +29,9 @@ module Amber
       end
 
       def draw(valve : Symbol, namespace : String)
+        scope = Scope.new
         scope.push(namespace)
         with DSL::Router.new(self, valve, scope) yield
-        scope.pop
```

Comment:
Why not poping this scope ?

Reply:
Because, It's using in another instance.

Reply:
But by using another instance, you erase previous scopes, like 
```
namespace "/foo" do
  namespace "/bar" do
  end
end
```

Here, `/foo` vanished.

Reply:
@alex-lairan I pushed new commits, that contain code modified with namespace.